test/unit/demutils.js

```diff
@@ -0,0 +1,93 @@
+import * as THREE from 'three';
+import ElevationLayer from 'Layer/ElevationLayer';
+import WMTSSource from 'Source/WMTSSource';
+import HttpsProxyAgent from 'https-proxy-agent';
+import assert from 'assert';
+import GlobeView from 'Core/Prefab/GlobeView';
+import Coordinates from 'Core/Geographic/Coordinates';
+import Extent from 'Core/Geographic/Extent';
+import { updateLayeredMaterialNodeElevation } from 'Process/LayeredMaterialNodeProcessing';
+import TileMesh from 'Core/TileMesh';
+import OBB from 'Renderer/OBB';
+import LayerUpdateState from 'Layer/LayerUpdateState';
+import DEMUtils from 'Utils/DEMUtils';
+import MaterialLayer from 'Renderer/MaterialLayer';
+import Renderer from './bootstrap';
+
+describe('DemUtils', function () {
+    const renderer = new Renderer();
+    const placement = { coord: new Coordinates('EPSG:4326', 1.5, 43), zoom: 10 };
+    const viewer = new GlobeView(renderer.domElement, placement, { renderer });
+
+    const source = new WMTSSource({
+        format: 'image/x-bil;bits=32',
+        projection: 'EPSG:4326',
+        url: 'https://wxs.ign.fr/3ht7xcw6f7nciopo16etuqp2/geoportail/wmts',
```

Comment:
Same, couldn't we have a local file to load ? That way we don't have to search for hours new values if there is any change.